undercloud.yml

```diff
@@ -70,3 +89,16 @@
             chdir: "{{ infrared_dir }}"
         changed_when: false
         when: osp_release|int >= 17
+
+      - name: Run the undercloud-ipa-install.yml playbook
+        shell: |
+          source /home/stack/.bash_profile
+          ansible-playbook --ssh-extra-args "-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null" /usr/share/ansible/tripleo-playbooks/undercloud-ipa-install.yaml
+        when: tls_everywhere == true
+        delegate_to: undercloud
+
+      - name: Update the undercloud for TLS-Everywhere
+        shell:
+          openstack undercloud install 2>&1 | tee undercloud_install.log
```

Comment:
can you add prefix `(time openstack undercloud install) &> undercloud_install_tls.log` as we want to capture the time taken to update undercloud